hw/xbox/nv2a/nv2a_psh.c

```diff
@@ -49,120 +49,128 @@
  */
 
 
-enum PS_TEXTUREMODES
-{                                 // valid in stage 0 1 2 3
-    PS_TEXTUREMODES_NONE=                 0x00L, // * * * *
-    PS_TEXTUREMODES_PROJECT2D=            0x01L, // * * * *
-    PS_TEXTUREMODES_PROJECT3D=            0x02L, // * * * *
-    PS_TEXTUREMODES_CUBEMAP=              0x03L, // * * * *
-    PS_TEXTUREMODES_PASSTHRU=             0x04L, // * * * *
-    PS_TEXTUREMODES_CLIPPLANE=            0x05L, // * * * *
-    PS_TEXTUREMODES_BUMPENVMAP=           0x06L, // - * * *
-    PS_TEXTUREMODES_BUMPENVMAP_LUM=       0x07L, // - * * *
-    PS_TEXTUREMODES_BRDF=                 0x08L, // - - * *
-    PS_TEXTUREMODES_DOT_ST=               0x09L, // - - * *
-    PS_TEXTUREMODES_DOT_ZW=               0x0aL, // - - * *
-    PS_TEXTUREMODES_DOT_RFLCT_DIFF=       0x0bL, // - - * -
-    PS_TEXTUREMODES_DOT_RFLCT_SPEC=       0x0cL, // - - - *
-    PS_TEXTUREMODES_DOT_STR_3D=           0x0dL, // - - - *
-    PS_TEXTUREMODES_DOT_STR_CUBE=         0x0eL, // - - - *
-    PS_TEXTUREMODES_DPNDNT_AR=            0x0fL, // - * * *
-    PS_TEXTUREMODES_DPNDNT_GB=            0x10L, // - * * *
-    PS_TEXTUREMODES_DOTPRODUCT=           0x11L, // - * * -
-    PS_TEXTUREMODES_DOT_RFLCT_SPEC_CONST= 0x12L, // - - - *
-    // 0x13-0x1f reserved
+enum PS_TEXTUREMODES { // valid in stage 0 1 2 3
```

Comment:
Bad alignment

*(This was brought up before, so I assume this comment is outdated I'll review a fixup in a few minutes - reviewing commit-by-commit right now)*